src/ClientData/include/ClientData/ApiTrackValue.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CLIENT_DATA_API_TRACK_VALUE_H_
+#define CLIENT_DATA_API_TRACK_VALUE_H_
+
+#include <cstdint>
+#include <string>
+#include <utility>
+
+namespace orbit_client_data {
+
+// Represents a value tracking event from our manual instrumentation API (Orbit.h). It contains a
+// name of the track to which the data point should be added, and the data point itself.
+// Note that we only display "double" values in the UI. Values of other types need to be converted
+// to double.
+// See `orbit_grpc_protos::ApiTrackValue*` protos.
+class ApiTrackValue {
```

Comment:
This was already like this and it matches the gRPC proto, so nothing to do I guess... but I just realized that I don't really understand the name. Is "track" in the sense of a track in the capture view, or in the sense of "api-tracked value" (value tracked with Orbit API)?

Reply:
It is the latter one. We "track" values.  Do you want me to rename it?

Reply:
Ok, it's grammatically a bit strange to have the verb in the base form. The proto is called the same so it's fine to leave it as it is for me. Unless you want to rename it, of course.

Reply:
Yeah, I agree. I will keep it as is for now, however.